lib/diagnostic.rb

```diff
@@ -49,6 +59,15 @@ def method_name
 
 # define your method here
 
+def odd(num)
+   if num % 2 == 1
+      return "#{numb} is odd"
+   end
+   if num % 2 == 0
+      return "#{num} is even"
+  end
+end
+
 Diagnostic.predicate = # call your method here
```

Comment:
numb vs num, check your variable names.  Also you didn't call it.  Why two separate if statements?
something like this:
```
def odd?(number)
  number % 2 == 1
end

Diagnostic.predicate = odd?(4)
```